spec/components/task_list/view_spec.rb

```diff
@@ -0,0 +1,68 @@
+require "rails_helper"
+
+RSpec.describe TaskList::View do
+  alias_method :component, :page
+
+  let(:status) { nil }
+
+  before(:each) do
+    render_inline(TaskList::View.new) do |component|
+      component.slot(
+        :row,
+        task_name: "some key",
+        path: "some_path",
+        status: status,
+      )
+    end
+  end
+
+  context "when task data is provided" do
+    context "rendered tasks" do
+      it "renders a list of tasks" do
+        expect(component).to have_selector(".app-task-list__item")
+      end
+
+      it "renders the task name" do
+        expect(component).to have_link("some key", href: "some_path")
+      end
+    end
+
+    context "tags" do
```

Comment:
Minor quibble: we are testing the `tags` here (I believe?), so probably should be `describe "tags"`.